configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
```

Comment:
@mandarjog @geeknoid : 

The plan is to create a oneof from all the possible templates, during the mixer build time, that would be part of the types array
message ServiceConfig {
  repeated Type types = 1;
  ..
}
message Type {
  string name = 1
  oneof Params {
    foo.bar.Metric metric_param = 2;
    foo.bar.Qupta quota_param = 3;
  }
}

@louiscryan  you are proposing we do something like:

message ServiceConfig {
  repeated QuotaType quota_types = 1;
  repeated MetricType metric_types = 2;
  ..
}
message QuotaType {
  string name = 1;
  // We don't need param's then. We can just flatten the fields of param's here
  map<string, istio.mixer.v1.config.descriptor.ValueType> dimensions = 2;

  int64 max_amount = 5;
  google.protobuf.Duration expiration = 6;
}
Thoughts from Mixers ?